CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.1.1 (2017-12-03)
```

Comment:
Not sure why GitHub show previous changes...

Reply:
I'll see if i can figure it out..
Learning GIT is not the easiest thing in the world, thats for sure.